core/src/main/java/hudson/model/Job.java

```diff
@@ -262,20 +268,6 @@ public void onCopied(Item src, Item item) {
         }
     }
 
-    @Override
-    protected void performDelete() throws IOException, InterruptedException {
```

Comment:
need to not hold the lock while interrupting or the interrupted threads may be unable to complete